core/main/src/Core/Canvas.ts

```diff
@@ -434,8 +435,26 @@ export class Canvas {
 
         this.drawParticleLinks(particle);
 
-        if (radius > 0) {
-            const zSizeFactor = 1 - zIndexOptions.sizeRate * particle.zIndexFactor;
+        const orbitOptions = particle.particlesOptions.orbit;
+        const zSizeFactor = 1 - zIndexOptions.sizeRate * particle.zIndexFactor;
+
+        if (radius > 0 && orbitOptions.enable === false) {
```

Comment:
The `drawParticle` code is duplicated and long, using two `if`s to check if the orbits need to be drawn is shorter and safer, and an enum instead of `"front"` and `"back"` should be a better solution